src/odemis/dataio/tiff.py

```diff
@@ -3133,3 +3142,101 @@ def _convert_tescan_to_odemis_metadata(metadata: bytes) -> dict:
     
     return md
 
+def _convert_openfibsem_to_odemis_metadata(metadata: str) -> dict:
+
+    # load json metadata
+    omd = json.loads(metadata)        
+    md = {}
+
+    # general
+    try:
+        timestamp = omd["microscope_state"]["timestamp"]
+        md[model.MD_ACQ_DATE] = timestamp
+    except Exception as e:
+        logging.warning(f"Failed to parse OpenFIBSEM metadata: {model.MD_ACQ_DATE} - {type(e)}: {e}")
+
+    try:
+        system_info = omd["system"]["info"]
+        md[model.MD_HW_NAME] = system_info["manufacturer"] + " " + system_info["model"] + "-" + system_info["serial_number"]  
+    except KeyError as e:
+        logging.warning(f"Failed to parse OpenFIBSEM metadata: {model.MD_HW_NAME} -  {type(e)}: {e}")
+    try:
```

Comment:
Please add an empty line before the `try`.

Reply:
empty line at before 3093 before the try